common/lib/xmodule/xmodule/templates/library-sourced-block-studio-view.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+<div class="editor-with-buttons">
+  <div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+    <div class="list-input settings-list">
+      {{ react_content|safe }}
+    </div>
+  </div>
+  <div class="xblock-actions">
+    <ul>
+      <li class="action-item">
+        <a href="#" class="button action-primary save-button" data-submit-url="{{save_url}}">{% trans "Save" as tmsg %} {{tmsg|force_escape}}</a>
```

Comment:
It seems that hitting "Save" here has the effect of re-importing all selected blocks from their libraries whether or not they have changed, grabbing from the latest library versions and overwriting any post-import changes to the blocks that the author may have made.

There's nothing wrong with that per se--in fact, I'd say it's the least complicated thing to do from a BD-14 development perspective. It might be quite unpleasantly surprising from a course author perspective, though.

In Milestone 3, we should explore the import semantics more thoughtfully. Maybe we will end up leveraging the draft numbers from Blockstore, so that course authors could select blocks from older library versions if they wanted to. Maybe we will try to apply any author's edits on top of the updated library blocks. We will also have to think hard about things like Grading and how they interact with updates to blocks. Some of these nuances have burned us recently with V1 Content Libraries.

For this PR, though, what you have works great, with the caveat that I wouldn't be ready for it to go to production. 

For now, could you please add some brief warning text inside the selection modal, informing the author that changes to previously imported blocks will be overwritten? Also consider changing the button text to something that sounds more destructive, e.g. "Save and Import". These changes will help Marco and others understand what's going on as they check this out on Stage, and it will help us remember the temporary decision that we've made here.

also cc'ing @arbrandes and @ormsbee here due to their potential interest in this.